tests/wpunit/Auth/Admin/ConnectControllerTest.php

```diff
@@ -54,47 +60,38 @@ protected function setUp(): void {
 		);
 	}
 
-	protected function tearDown(): void {
-		$GLOBALS['current_screen'] = null;
-
-		parent::tearDown();
-	}
-
 	public function test_it_stores_basic_token_data(): void {
 		global $_GET;
 
 		wp_set_current_user( 1 );
 
-		$this->assertNull( $this->token_manager->get() );
+		$this->assertNull( $this->token_manager->get( $this->plugin ) );
```

Comment:
I think accepting a non nullable Resource, we lost backwards compatibility. If we are ok with that, we can ignore my related comments about it.

Reply:
Yeah we are good here, Kadence isn't using this instance directly but an interface to it that takes a slug and converts it to a resource already, but their implementation is still a bit different and I'll sort that out after.

Reply:
Perfect! im resolving all my related comments about this! Thank you